recipes/doctr-versions-menu/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "doctr-versions-menu" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```

Comment:
`url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/doctr_versions_menu-{{ version }}.tar.gz`

Reply:
Thank you! Addressed via https://github.com/conda-forge/staged-recipes/pull/12250/commits/1f000c20219b59e174dc8bc0e336720c6728c53d